testIntegration/tests/authHelper.test.js

```diff
@@ -26,15 +26,15 @@ describe("JS: Auth Helper getShopperToken", () => {
         {
           authType: "guest",
           customerId: "ab1avTqmPIwc4MRq0cEs59LEka",
-          preferredLocale: "en_US"
+          preferredLocale: "en_US",
```

Comment:
why "," ?

Reply:
A recent update to eslint changed our comma dangle rule. They're now required.